### DISCUSSION TITLE
使用scanf导致满屏警告，我该怎么办

### DISCUSSION THREAD

**Discussion by VansamaZWT** [liusy58/NJU_AdvancedProgramming_SP22#86] [Q&A] [answered]
以前使用的输入都是cin，这次为了输入安全使用scanf，却导致一直发生如下两则警告，我该怎么办？
![)ZTB@I@QK@O4}9TG_`XGCIH](https://user-images.githubusercontent.com/100075314/156122210-2ee3f7aa-ba27-449c-9da3-0a7b938876c7.png)

**Comment by bliss-bug** [comment]
scanf的返回值是int，而不是void，所以没有记录返回值系统会有warning，但只要没有错误应该没问题

**Comment by RicoloveFeng** [comment, marked as answer]
关闭Visual Studio的SDL检查

**Comment by RicoloveFeng** [comment]
另外，请记得善用搜索引擎：

https://cn.bing.com/search?q=%E8%BF%94%E5%9B%9E%E5%80%BC%E8%A2%AB%E5%BF%BD%E7%95%A5+scanf
